<ui version="4.0" >
 <class>BookExportDialog</class>
 <widget class="QDialog" name="BookExportDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Export Font Book</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QListWidget" name="templatesList" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="selectedFontsList" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="label" >
           <property name="text" >
            <string>Output</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="fileNameEdit" />
         </item>
         <item>
          <widget class="QPushButton" name="fileNameButton" >
           <property name="text" >
            <string>Browse...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGraphicsView" name="templatePreview" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
         <horstretch>2</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <widget class="QDialogButtonBox" name="okButton" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
